pennylane/_device.py

```diff
@@ -696,7 +699,7 @@ def expand_fn(self, circuit, max_expansion=10):
 
         return self.default_expand_fn(circuit, max_expansion=max_expansion)
 
-    def batch_transform(self, circuit):
+    def batch_transform(self, circuit: QuantumTape):
```

Comment:
Should we already call it QuantumScript here?

Reply:
Yes! Changed